src/coreclr/System.Private.CoreLib/Tools/GenUnicodeProp/TableLevels.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace GenUnicodeProp
 {
-    internal class TableLevels
+    internal sealed class TableLevels
```

Comment:
Heads up that files like this aren't shipping code. This change is ok, but we'll want to make sure the analyzer rule is scoped only to shipping assemblies.

Reply:
Yup